privacy-protections/storage-partitioning/main.js

```diff
@@ -252,8 +252,8 @@ if (window.location.hostname === 'localhost') {
     runButton.setAttribute('disabled', '');
 }
 
-// warn the user if loaded over privacy-test-pages.glitch.me
-if (window.location.hostname === 'privacy-test-pages.glitch.me') {
```

Comment:
some of those changes mean that `privacy-test-pages.glitch.me` will no longer work fully as expected - here it's not a big deal but just FYI

Reply:
Closing - we agreed that we will decommission glitch.me